app/src/main/java/org/wikipedia/notifications/NotificationLinkHandler.kt

```diff
@@ -11,7 +11,9 @@ import org.wikipedia.page.PageTitle
 import org.wikipedia.util.CustomTabsUtil
 import org.wikipedia.util.log.L
 
-class NotificationLinkHandler constructor(context: Context) : LinkHandler(context) {
+class NotificationLinkHandler(context: Context) : LinkHandler(context) {
+
+    var category: NotificationCategory? = null
```

Comment:
Can we find a way to make the `category` be part of the constructor of `NotificationLinkHandler`?

Reply:
Yes!